src/otx/v2/adapters/torch/mmengine/mmseg/modules/models/utils/asymmetric_position_attention.py

```diff
@@ -0,0 +1,122 @@
+"""Asymmetric position attention module."""
+# Copyright (c) 2019 MendelXu
+# SPDX-License-Identifier: Apache-2.0
+#
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+from __future__ import annotations
+
+import torch
+import torch.nn.functional
+from mmcv.cnn import ConvModule
+from torch import nn
+
+from .psp_layer import PSPModule
+
+
+# pylint: disable=too-many-instance-attributes
+class AsymmetricPositionAttentionModule(nn.Module):
+    """AsymmetricPositionAttentionModule.
+
+    Reference: https://github.com/MendelXu/ANN.
+    """
+
+    def __init__(
+        self,
+        in_channels: int,
+        key_channels: int,
+        value_channels: int | None = None,
+        psp_size: tuple[int, ...] | None = None,
```

Comment:
This is one of variables for PyramidSceneParsing (PSP Module). I'd like to keep the name, but I changed `class PSPModule` to `class PyramidSceneParsing`, so I think it should be quite clear to call it `psp_size` instead of `pyramid_scene_parsing_pooling_size ` (as it's a bit verbose I think)

Reply:
fair comment